app/src/actions/user.js

```diff
@@ -57,7 +57,7 @@ export function logout() {
       type: LOGOUT
     });
     window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
-    if (window.location.pathname.match('^\/map')) {
+    if (window.location.pathname.match('/map')) {
```

Comment:
A few notes:
- in regexes forward slashes need to be escaped (although some regex engines might work in different ways)
- '\\/map' should translate to  '\/map', so not what we are looking for here.
- if you have different results on lint validation, please check your node versions